resources/services/ec2/instance_statuses.go

```diff
@@ -0,0 +1,163 @@
+package ec2
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func Ec2InstanceStatuses() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_ec2_instance_statuses",
+		Description:  "Describes the status of an instance.",
+		Resolver:     fetchEc2InstanceStatuses,
```

Comment:
You should have a PK on each table. For this table you can use the ARN field (see comment below)